src/gamefalloutttw.cpp

```diff
@@ -127,7 +127,7 @@ bool GameFalloutTTW::isInstalled() const
 
 QString GameFalloutTTW::gameName() const
 {
-  return "TTW";
+    return "Tale of Two Wastelands";
```

Comment:
Might need confirmation but that could break many existing TTW instance, so that would be a very bad idea in this case.

Reply:
It would indeed, I just tested it

Hmm, is there anyway we could migrate the gameName to something new?

Reply:
Unfortunately no, there is no possible way to do that... Even if we could migrate the game name when upgrading MO2, we probably would not want to do that because we want user to be able to downgrade MO2 if they need to.

What are the places where you'd prefer to see "Tale of Two Wastelands" instead of "TTW"?

Reply:
The label of the MO2 window in the top left (included a pic), and the initial profile game selection. Perhaps a new plugin base function `displayGameName()` could be added that's used for both those places, defaulting to the `gameName()`?

<img width="223" alt="ModOrganizer_HOS2RC7klO" src="https://github.com/ModOrganizer2/modorganizer-game_ttw/assets/5809130/b3540d1b-912b-42d0-be14-6e16bf04c23e">

Reply:
Yes, that's probably the way to go. That would also allow plugin to provide localized display names for games which is currently not possible.

Reply:
I'll go that route, changing this PR to a draft in the meantime